src/lidar_tools/dsm_functions.py

```diff
@@ -799,7 +1014,7 @@ def gdal_warp(
     src_srs: str,
     dst_srs: str,
     res: float = 1.0,
-    resampling_alogrithm: str = "cubic",
+    resampling_alogrithm: str = "bilinear"
```

Comment:
i typically use bilinear as default, but not sure if cubic was intentional from earlier

Reply:
Yes, it was old legacy option. I checked all the operators when we were having tiling based seam errors (fixed now), and during that I found bilinear to be the best! So I switched it to bilinear recently.